articles/vmware/vmw-faq.md

```diff
@@ -552,13 +541,15 @@ The recovery time objective (RTO) depends on the nature of the disaster or failu
 
 If you need more control and assurance around how data is replicated, the DR service is tested, or the solution handles failover and failback, we recommend you create your own DR solution by using independent sets of VMs in the various regions offered by the UKCloud platform.
 
+An alternative UKCloud solution to acheive DR with a low RPO and RTO is [Journaling Protection](vmw-sco-journaling-protection.md).
+
 ### How does UKCloud enable customers to create their own DR solutions?
 
-Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or in the application.
+Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or into the application.
```

Comment:
Issue existed in the original text but while you're editing, could you change customers services to customers' services (add apostrophe)